buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1058,6 +1058,30 @@ def set_job_variables(job_type) {
     USE_TESTENV_PROPERTIES = params.USE_TESTENV_PROPERTIES ?: false
     echo "Using USE_TESTENV_PROPERTIES = ${USE_TESTENV_PROPERTIES}"
 
+    if (PERSONAL_BUILD && PERSONAL_BUILD.equalsIgnoreCase('true')) {
+        LIGHT_WEIGHT_CHECKOUT = true
+        GENERATE_JOBS = params.AUTOMATIC_GENERATION ?: false
```

Comment:
I think we need to always generate. We don't know what ran last. If user has default params and previous run was a personal repo, we need to put it back to default.

Reply:
AUTOMATIC_GENERATION is true by default. It means GENERATE_JOBS is true (always generate the test job).

Reply:
Ah good point. So, shouldn't the code be `GENERATE_JOBS = params.AUTOMATIC_GENERATION ?: true` then? Why would we default to false if it is not defined?

Reply:
updated